.devcontainer/Dockerfile

```diff
@@ -25,6 +25,70 @@ RUN chmod -R 555 /data/
 
 FROM ${DEVCONTAINER_BASE} AS devcontainer
 
+#########################################################
```

Comment:
Moving the MATLAB stuff to an earlier layer because it doesn't change very often.